kmp-library/src/commonMain/kotlin/com/novoda/aws/v4/signer/PathUtil.kt

```diff
@@ -0,0 +1,296 @@
+package com.novoda.aws.v4.signer
+
+private const val IGNORED = -1
+
+internal object PathUtil {
```

Comment:
Should we add a comment where this is coming from `new URI("http://" + encoded).normalize()`?